tahoe_sites/api.py

```diff
@@ -378,6 +378,40 @@ def is_exist_organization_user_by_email(email, organization, must_be_active=Fals
     return user is not None
 
 
+def get_tahoe_sites_auth_backends():
+    """
+    Get a list of auth backends by tahoe_sites, considering the use of upstream edx-organizations or the forked repo
+
+    :return: a list of strings representing auth backends
+    """
+    package_name = 'organizations' if zd_helpers.should_site_use_org_models() else 'tahoe_sites'
+    return [
+        '{package_name}.backends.DefaultSiteBackend'.format(package_name=package_name),
+        '{package_name}.backends.OrganizationMemberBackend'.format(package_name=package_name),
+    ]
+
+
+def get_organizations_queryset_from_uuids(uuids):
```

Comment:
We always return QuerySets when possible, right?

```suggestion
def get_organizations_by_uuids(uuids):
```